docs/trace/customizing-the-sdk/README.md

```diff
@@ -446,45 +446,34 @@ it is shutdown.
 `TracerProvider`. Only a single `TraceProvider` may exist in an
 `IServiceCollection` \ `IServiceProvider`.
 
-### Dependency injection `TracerProviderBuilder` extension method reference
+### Dependency injection TracerProviderBuilder extension method reference
 
 * `AddInstrumentation<T>`: Adds instrumentation of type `T` into the
   `TracerProvider`.
 
+* `AddInstrumentation<T>(Func<IServiceProvider, T> instrumentationFactory)`:
+  Adds instrumentation of type `T` into the
+  `TracerProvider` using a factory function to create the instrumentation
+  instance.
+
 * `AddProcessor<T>`: Adds a processor of type `T` (must derive from
   `BaseProcessor<Activity>`) into the `TracerProvider`.
 
-* `SetSampler<T>`: Register type `T` (must derive from `Sampler`) as the sampler
-  for the `TracerProvider`.
+* `AddProcessor(Func<IServiceProvider, BaseProcessor<Activity>>
+  implementationFactory)`: Adds a processor into the `TracerProvider` using a
+  factory function to create the processor instance.
 
 * `ConfigureServices`: Registers a callback function for configuring the
   `IServiceCollection` used by the `TracerProviderBuilder`. **Note:**
   `ConfigureServices` may only be called before the `IServiceProvider` has been
-  created after which point service can no longer be added.
-
-* `ConfigureBuilder`: Registers a callback function for configuring the
```

Comment:
Note: You can still do "ConfigureBuilder" work by casting to `IDeferredTracerProviderBuilder` but I didn't mention that in the guide. I don't think it will be common for users to need to do that with the new APIs. Anyone feel like we should mention it? We could always add it if a common use case emerges.

Reply:
agree to skip mentioning it now.